scripts/categorizer.py

```diff
@@ -24,22 +93,27 @@ def main():
         lines = fp.readlines()
 
     empty_lines = []
-    comment_lines = []
     assigment_lines = []
     conditional_lines = []
     control_lines = []
     indent_levels = []
     function_call_lines = []
     function_def_lines = []
 
+    # Categorize solely comment lines
+    comment_lines = find_comment_lines(lines)
+    # Sanatize the code from comments
+    strip_comments_after_code(lines, comment_lines)
+    # Get rid of comments on same lines as code
+
+
     # Categorizing lines
     for line in lines:
         empty_lines.append(find_empty_lines(line))
 
-
     # Debug portion
-    print("Empty Lines")
-    print_all_cats(lines, empty_lines)
+    print("Comment Lines")
```

Comment:
Find a more genric way to do this, perhaps just print out the variable name as it is descriptive enough.

Reply:
Nah not now.